app/src/app/Marketplace.jsx

```diff
@@ -9,7 +9,7 @@ import routes from '$routes'
 import links from '../links'
 import { formatPath } from '$shared/utils/url'
 
-import GenericErrorPage from '$shared/components/GenericErrorPage'
+import ErrorPage from '$shared/components/ErrorPage'
```

Comment:
`ErrorPage` is responsible for rendering proper page-like error component for given exception.